Sources/tart/Commands/Run.swift

```diff
@@ -112,7 +112,23 @@ struct Run: AsyncParsableCommand {
 
   @MainActor
   func run() async throws {
-    let vmDir = try VMStorageLocal().open(name)
+    let localStorage = VMStorageLocal()
+    let vmDir = try localStorage.open(name)
+
+    let storageLock = try FileLock(lockURL: localStorage.baseURL)
```

Comment:
Is there a reason to lock `localStorage.baseURL` instead of `Config().tartHomeDir` here?

Take a look at `Import.swift` and `Clone.swift` — they both use `Config().tartHomeDir` when modifying a list of VMs/checking for duplicated MAC-addresses.